TodoList/bot/api/graphClient.ts

```diff
@@ -0,0 +1,38 @@
+// Import polyfills for fetch required by msgraph-sdk-javascript.
+import "isomorphic-fetch";
+import { TeamsFx, createMicrosoftGraphClient, IdentityType } from "@microsoft/teamsfx";
+import { Client, ResponseType }from "@microsoft/microsoft-graph-client"
+
+interface GraphResponse {
+    status: number;
+    body: { [key: string]: any };
+}
+
+export default async function graphRun(AADId: string, token: string) {
+    // Initialize response.
+    const res: GraphResponse = {
+        status: 200,
+        body: {},
+    };
+    if (!token || !token.trim()) {
+        throw new Error('SimpleGraphClient: Invalid token received.');
```

Comment:
Can change error message to `Empty token received` to better describe the problem